authentication/src/main/java/io/camunda/authentication/config/WebSecurityConfig.java

```diff
@@ -77,6 +77,7 @@ public SecurityFilterChain securityFilterChain(final HttpSecurity httpSecurity)
     try {
       return httpSecurity.build();
     } catch (final Exception e) {
+      // return httpSecurity.getObject();
```

Comment:
Just to see what happens, but this is where the `auth` module is building. Optimize is clashing with this